src/wp-includes/class-wp-metadata-lazyloader.php

```diff
@@ -133,15 +133,19 @@ public function reset_queue( $object_type ) {
 	 * @since 4.5.0
```

Comment:
Please add another `@since` annotation below this to outline the addition of the new parameter.